region/maridia/inner-pink/Draygon's Room.json

```diff
@@ -246,19 +246,36 @@
           "usedTiles": 22,
           "openEnd": 0
         }},
-        {"enemyDamage": {
-          "enemy": "Draygon",
-          "type": "contact",
-          "hits": 2
-        }},
         {"or": [
-          "HiJump",
+          "Morph",
+          {"enemyDamage": {
+            "enemy": "Draygon",
+            "type": "contact",
+            "hits": 1
+          }}
+        ]},
+        {"or": [
+          "canTrickyJump",
           {"enemyDamage": {
             "enemy": "Draygon",
             "type": "contact",
             "hits": 1
           }}
         ]},
+        {"or": [
+          "HiJump",
+          {"enemyDamage": {
+            "enemy": "Draygon",
+            "type": "contact",
+            "hits": 1
+          }},
+          "h_pauseAbuseMinimalReserveRefill"
```

Comment:
I don't think these is practical since you need high RegularEnergy throughout the fight to shinespark.  Pauseabusing can be used in other strats to walk through Draygon after a goop instead of dodging, or to escape a grab, and a morphless suitless doesn't sound impossible with it.